gcp_variant_transforms/testing/vcf_header_util.py

```diff
@@ -16,28 +16,17 @@
 
 from __future__ import absolute_import
 
-from vcf import parser
-
 from gcp_variant_transforms.beam_io import vcf_header_io
 
 
 def make_header(header_num_dict):
   # type: (Dict[str, str]) -> VcfHeader
   """Builds a VcfHeader based on the header_num_dict.
 
-  All fields of parser._Info are set to their default values except for the
-  'id' which is set to the keys in header_num_dict and 'num' which is set based
-  on header_num_dict values mapped according to parser.field_counts.
-
   Args:
     header_num_dict: a dictionary mapping info keys to string num values.
   """
   infos = {}
   for k, v in header_num_dict.iteritems():
-    if v in parser.field_counts:
-      pyvcf_num_field_value = parser.field_counts[v]
-    else:
-      pyvcf_num_field_value = int(v)
```

Comment:
What is the type of `v`?
Don't we need to follow the old logic and convert it to int? Here is what I think it should be:
```
num_field_value = v if v in HEADER_NUMBERS else int(v)
infos[k] = vcf_header_io.CreateInfoField(k, num_field_value, '.', '')
```